doc/sphinx/source/quickstart/installation.rst

```diff
@@ -35,16 +35,16 @@ available with more explanations and guidance if the installation instructions
 here are too concise.
 See `common installation issues`_ if you run into trouble.
 
-.. _install_with_conda:
+.. _install_with_mamba:
 
-Conda installation
+Mamba installation
 ==================
 
 In order to install the `conda <https://docs.conda.io>`_ package, you will need
-conda pre-installed.
+mamba pre-installed.
```

Comment:
this needs a bit of TLC: we should say one needs anaconda first (conda) then they need to install `mamba` in the `(base)` conda env with `conda install -c conda-forge mamba`